packages/toolkit/utils/src/cli/is/project.ts

```diff
@@ -139,7 +143,7 @@ export const isReact18 = (cwd: string = process.cwd()) => {
     return false;
   }
 
-  return semver.satisfies(semver.minVersion(deps.react)!, '>=18.0.0');
+  return semver.gte(semver.minVersion(deps.react)!, '18.0.0');
```

Comment:
why change satisfies to gte.

Reply:
The `satisfies` function can't verify the test version well, I encountered problem when I was using react@canary, so I modified this.